src/bin/cargo/commands/fix.rs

```diff
@@ -125,6 +131,20 @@ pub fn exec(config: &mut Config, args: &ArgMatches<'_>) -> CliResult {
     // code as we can.
     let mut opts = args.compile_options(config, mode, Some(&ws))?;
 
+    let clippy_args = args
+        .value_of("clippy") // always yields None
```

Comment:
This no longer works, I cant figure out how to make `--clippy` work by itself without args and also let it optionally take args.